### PR TITLE
Add integration with ajv-errors

### LINKED ISSUE #139: Integration with ajv-errors (or otherwise support custom error messages)
I am an owner of the MDN browser-compat-data project, and we use `ajv` with `better-ajv-errors`.  As indicated in https://github.com/mdn/browser-compat-data/issues/3256, we are wanting to use `ajv-errors` for custom errors to better represent the situation where a subfeature contains invalid characters (which is caught by `additionalProperties: false`).  Doing a quick experiment, however, I found that it seems that `better-ajv-errors` doesn't get along with `ajv-errors`.  It seems that `ajv-errors` replaces the error type for some odd reason.

It would be really wonderful to see support included so we can use both at the same time.  Happy to spend time to do this one!

### PR DESCRIPTION
**Pull request by queengooborg**
This PR fixes #139 by remapping the custom messages from `ajv-errors`.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -20,7 +20,14 @@ const isXError = x => (error /*: Error */) => error.keyword === x;
 export const isRequiredError = isXError('required');
 export const isAnyOfError = isXError('anyOf');
 export const isEnumError = isXError('enum');
-export const getErrors = (node /*: Node*/) => (node && node.errors) || [];
+export const getErrors = (node /*: Node*/) =>
+  node && node.errors
+    ? node.errors.map(e =>
+        e.keyword === 'errorMessage'
+          ? { ...e.params.errors[0], message: e.message }
```
Comment (torifat) [inline src/utils.js:27]: Can we just ignore the rest of the errors?

**Comment by torifat** [comment]
I also added a changeset, so please pull if you want to make any further changes.

File: src/utils.js (hunk 1 above)
Comment (queengooborg) [reply src/utils.js:27]: I tested a number of different scenarios and found that each `errorMessage` type only contains one error.  Testing included triggering both `additionalProperties` and `type`, with a custom error message for everything then for each type, and finally turning on the `singleError` option when initializing `ajv-errors`.

**Comment by torifat** [comment]
Thank you, I will merge your changes soon. Then the CI will release a new minor version.

**Pull request merged**